packages/neon/neon/lib/src/widgets/image.dart

```diff
@@ -257,10 +216,10 @@ class NeonApiImage extends StatelessWidget {
     super.key,
   }) : account = null;
 
-  /// Creates a new Neon API image for a given account.
+  /// Creates a new Neon API image with the given [account].
   ///
-  /// Use [NeonApiImage] to fetch the image with the currently active account.
-  const NeonApiImage.custom({
+  /// See [NeonApiImage] for using the active account.
```

Comment:
```suggestion
  /// Creates a new Neon API image fetching the image with the given [account].
  ///
  /// See [NeonApiImage] to fetch the image using the currently active account.
```

We shouldn't shorten the message as the message should be clear and precise on its own.